scripts/Vulkan.py

```diff
@@ -10,12 +10,14 @@
 from zipfile import ZipFile
 
 VULKAN_SDK = os.environ.get('VULKAN_SDK')
-VULKAN_SDK_INSTALLER_URL = 'https://sdk.lunarg.com/sdk/download/1.2.170.0/windows/vulkan_sdk.exe'
+VULKAN_SDK_INSTALLER_URL = f"https://sdk.lunarg.com/sdk/download/{HAZEL_VULKAN_VERSION}/windows/VulkanSDK-{HAZEL_VULKAN_VERSION}-Installer.exe"
 HAZEL_VULKAN_VERSION = '1.2.170.0'
+VULKAN_SDK_EXE_DIR = 'Hazel/vendor/VulkanSDK'
 VULKAN_SDK_EXE_PATH = 'Hazel/vendor/VulkanSDK/VulkanSDK.exe'
 
 def InstallVulkanSDK():
     print('Downloading {} to {}'.format(VULKAN_SDK_INSTALLER_URL, VULKAN_SDK_EXE_PATH))
+    os.makedirs(VULKAN_SDK_EXE_DIR, exist_ok=True)
```

Comment:
Derive `VULKAN_SDK_EXE_DIR` from `VULKAN_SDK_EXE_PATH`
```suggestion
VULKAN_SDK_EXE_PATH = 'Hazel/vendor/VulkanSDK/VulkanSDK.exe'

def InstallVulkanSDK():
    print('Downloading {} to {}'.format(VULKAN_SDK_INSTALLER_URL, VULKAN_SDK_EXE_PATH))
    os.makedirs(os.path.dirname(VULKAN_SDK_EXE_PATH), exist_ok=True)
```